scripts/core/kernel.yml

```diff
@@ -352,7 +352,6 @@ params:
       desc: "[in][optional] Pointer obtained by USM allocation or virtual memory mapping operation. If null then argument value is considered null."
 returns:
     - $X_RESULT_ERROR_INVALID_KERNEL_ARGUMENT_INDEX
-    - $X_RESULT_ERROR_INVALID_KERNEL_ARGUMENT_SIZE
```

Comment:
You need to run the `generate` target to actually make the change to the generated files.

Reply:
Yes - done.